Backpack-SwiftUI/InsetBanner/Classes/README.md

```diff
@@ -31,7 +31,7 @@ BPKSponsoredBanner(
 The info icon can be disabled passing the `showIcon` value in `callToAction`: 
 
 ```swift 
-BPKSponsoredBanner(
+BPKInsetBanner(
     title: "Title",
     subheadline: "Subheading",
     callToAction: .init(
```

Comment:
for the CTA, should we actually allow hiding the (i) icon, but setting an a11y label?

Reply:
Intersting, indeed I don't see a point to the `accessibilityHint` if `showIcon` is false

Reply:
That is indeed how it works now, unless I am missing something?